include/SDL3/SDL_gpu.h

```diff
@@ -44,49 +44,174 @@ extern "C" {
 
 /* Type Declarations */
 
+/**
+ * An opaque handle representing the SDL_GPU context.
+ *
+ * \since This struct is available since SDL 3.0.0
+ */
 typedef struct SDL_GPUDevice SDL_GPUDevice;
+
+/**
+ * An opaque handle representing a buffer.
+ * Used for vertices, indices, indirect draw commands, and general compute data.
```

Comment:
There should be a blank comment line after the brief title, separating remarks from the header.

Reply:
e.g.
```
/**
 * The flags on a window.
 *
 * These cover a lot of true/false, or on/off, window state. Some of it is
 * immutable after being set through SDL_CreateWindow(), some of it can be
 * changed on existing windows by the app, and some of it might be altered by
 * the user or system outside of the app's control.
 *
 * \since This datatype is available since SDL 3.0.0.
 *
 * \sa SDL_GetWindowFlags
 */
```